public/style.css

```diff
@@ -226,6 +135,175 @@ section {
   color: white;
 }
 
+/* --- Navigatie menu --- */
+
+nav {
+  position: fixed;
+  bottom: 5rem;
+  right: 5rem;
+  z-index: 100;
+
+  ul {
+    position: relative;
+    bottom: 4.5rem;
+    right: 2rem;
+    height: auto;
+    background: none;
+    transition: none;
+    
+    li {
+      position: absolute;
+      top: 1.5rem;
+      left: 1.5rem;
+      opacity: 0;
+      transition: all 0.3s;
+      list-style: none;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+
+      a {
+        display: inline-block;
+        padding: 0.75rem;
+        border-radius: var(--radius-1);
+        margin-bottom: 0.25rem;
+      }
+
+      img {
+        height: 2rem;
+        width: 2rem;
+      }
+
+      p {
+        font: 400 1rem var(--primary-font);
+        color: white;
+      }
+    }
+  }
+}
+
+.menu-button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  bottom: 4.5rem;
+  right: 2rem;
+  width: 3.5rem;
+  height: 3.5rem;
+  appearance: none;
+  border: none;
+  background-color: var(--menu-layover);
+  border-radius: var(--radius-1);
+  cursor: pointer;
+}
+
+.menu-button span {
+	position:absolute;
+  width:1.5em;
+	height:.2em;
+	border-radius: var(--radius-2);
+  background-color:white;
+	transition:.3s;
+}
+
+.menu-button span:nth-of-type(1) {
+	transform:translateY(-.6em);
+}
+
+.menu-button span:nth-of-type(3) {
+	transform:translateY(.6em);
+}
+
+.menu-open { 
+  ul {
+      display: block;
+      position: relative;
+  }
+  
+   ul li {
+    opacity: 1;
+  }
+  
+  ul li:first-child {
+    top: -4.5rem;
+    left: 2rem;
+  }
+  
+  ul li:nth-child(2) {
+    top: -7.25rem;
+    left: -3.25rem;
+  }
+  
+  ul li:nth-child(3) {
+    top: -4.5rem;
+    left: -8.5rem;
+  }
+  
+  ul li:nth-child(4) {
+    top: 1.75rem;
+    left: -8.5rem;
+  }
+
+  ul li:nth-child(5) {
+    top: 3.5rem;
+    left: -3.25rem;
+  }
+
+  &::before {
+    content: "";
+    position: fixed;
+    bottom: 0;
+    right: 0;
+    width: 20rem;
+    height: 20rem;
+    background: var(--menu-layover); 
+    border-top-left-radius: 70%;
+  }
+
+  .menu-button {
+    background: white;
+
+    span {
+      background: var(--menu-layover);
+    }
+  }
+
+  .menu-button span:nth-of-type(1) {
+    transform:rotate(45deg) scaleX(1.2);
+  }
+
+  .menu-button span:nth-of-type(2) {
+    transform:scaleX(0);
+  }
+  
+  .menu-button span:nth-of-type(3) {
+    transform:rotate(-45deg) scaleX(1.2);
+  }  
+}
+
+.menu li:first-child a {
+  background: var(--menu-1);
+}
```

Comment:
Hier zou je efficiënter gebruik kunnen maken van custom properties!

Google: '_Don't use custom properties for constants only_'